src/main/java/frc/robot/commands/TankDrive.java

```diff
@@ -0,0 +1,79 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.Constants;
+import frc.robot.IO;
+import frc.robot.subsystems.DriveTrain;
+
+public class TankDrive extends CommandBase {
+  private final DriveTrain m_drivetrain;
+  private final IO m_io;
+
+  /** Creates a new TankDrive. */
+  public TankDrive(DriveTrain drivetrain, IO io) {
+    // Use addRequirements() here to declare subsystem dependencies.
+    m_drivetrain = drivetrain;
+    m_io = io;
+    addRequirements(m_drivetrain);
+
+  }
+
+  // Called when the command is initially scheduled.
+  @Override
+  public void initialize() {
+
+
+  }
+    public void GTADrive(double leftTrigger, double rightTrigger, double turn) {
+        
+      if (-Constants.kJoystickTurnDeadzone <= turn
+            && turn <= Constants.kJoystickTurnDeadzone) {
+              turn = 0.0;
+        }
+
+        turn = Math.abs(turn) < Constants.kJoystickTurnDeadzone
```

Comment:
do we need the first part of the ternary here? it looks like that case is handled by the `if` above and we might only need the second part of this ternary if `turn` is greater than `0`?

Reply:
I removed the ternary and the if statement for simplicity.